### ISSUE TITLE
static black cards

### ISSUE THREAD

**Issue by n30cr0n** [jakelauer/allbadcards#203]
Hi, awesome site and awesome game. One thing that has come up with playing with friends is that it would be good to have a static black card when you’re picking yourself white answer card. In that I mean that when you’re on a mobile device (phone etc. ) and are picking your white card to play, have the black card static at the top so you can glance back at it. Have been many times where we have had to scroll back to the top to re-read the black card  before choosing our answer (granted the problem exacerbates with the increase in alcohol cosumption haha). Anyway, just something that might help along the development of this amazing project. Thanks for all you hard work. Stay safe.

**Comment by dennmuel** [comment]
I agree, a black card that travels along as you scroll would make for a better user experience. (This issue kind of rings a bell, but I didn't find a duplicate so maybe I just had the same idea sometime. :D) I'm only worried about the screen real estate on small devices. Other than that great idea, thank you!

**Comment by n30cr0n** [comment]
for the most part I don’t think it would impact the screen layout too much. One idea is that after you start to scroll and the black card disappears from the top of the screen, a smaller black banner locked to the top of the screen with the back card text in is still visible. Almost like the old iFrame layout of yesteryear web design. Upon scrolling back up to the top the small banner would transition back to the original black card layout. This, in theory should not impact the screen real estate on smaller devices to the point of being unplayable.

**Comment by dennmuel** [comment]
Sounds good to me :)

**Comment by jakelauer** [comment]
I have a plan for this that involves locking the black card text to the bottom of the screen when you scroll past it using an alternative layout. Btw, the dupe for this is #159 but I think this issue is better than that one, so I'll close it.
